ibm/config.go

```diff
@@ -1721,6 +1731,16 @@ func (c *Config) ClientSession() (interface{}, error) {
 		session.secretsManagerClientErr = fmt.Errorf("Error occurred while configuring IBM Cloud Secrets Manager API service: %q", err)
 	}
 
+	kubernetesServiceV1Options := &sat.KubernetesServiceApiV1Options{
+		URL:           envFallBack([]string{"IBMCLOUD_SATELLITE_API_ENDPOINT"}, "https://containers.cloud.ibm.com/global"),
```

Comment:
instead of `"https://containers.cloud.ibm.com/global"` use default url from sdk some thing like `sat.DefaultSeviceURL`

Reply:
Updated config.go to use DefaultSeviceURL